src/Config/ObjectModel/RuntimeConfig.cs

```diff
@@ -327,6 +327,17 @@ public bool CheckDataSourceExists(string dataSourceName)
         return _dataSourceNameToDataSource.ContainsKey(dataSourceName);
     }
 
+    /// <summary>
+    /// Get the default datasource name.
+    /// </summary>
+    /// <returns>default datasourceName.</returns>
+#pragma warning disable CA1024 // Use properties where appropriate. Reason: Do not want datasource serialized and want to keep it private to restrict set;
+    public string GetDefaultDataSourceName()
+#pragma warning restore CA1024 // Use properties where appropriate
```

Comment:
changing that field to private so we can just internally set it, and then still use the getter here.

Reply:
Can you utilize this syntax so that we don't have to ignore StyleCop warning? https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/using-properties . To address your comment "Do not want datasource serialized" wouldn't adding `[JsonIgnore]` mitigate?

Do any of the exceptions in the C1024 article apply here? https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1024
if so, can the exception text be added to your comment?

Reply:
Yes, I can refactor this, but just to be clear, this was already existing code I originally removed because I was going to make that field public, then reverted.

Reply:
thanks for cleaning this up. The getter and setter are applicable to what you're working on so it's helpful to have made this change.